src/ex3/commands.js

```diff
@@ -0,0 +1,194 @@
+// imports
+import fs from 'fs';
+import { PokemonClient } from './pokemonClient.js';
+import { ItemManager } from './itemManager.js';
+import chalk from 'chalk';
+
+// PokemonClient instance
+const pokemonClient = new PokemonClient();
+const itemManager = new ItemManager();
+
+/* Add todo  function */
+const addTodo = async (todo) => {
+    let todoText = '';
+    let message = '';
+    if(itemManager.isPokemon(todo)){
+        const pokemon = await pokemonClient.getPokemonName(todo);
+        const type = await pokemonClient.getPokemonsTypes(pokemon);
+        todoText =`Catch ${pokemon}, a ${type} pokemon`;
+        message = `Catch ${chalk.bold(pokemon)}, a ${type} pokemon`;
+    } else{
+        todoText = todo;
+        message = "New todo added successfully!";
+    }
+    try{
+        fs.access('todos.txt', (err) => {
+            if (err){ 
+                // if todos.txt doesn't exist then create it
+                fs.writeFileSync('todos.txt', todoText);
+                console.log(chalk.bgGreen(message));
+            } else{
+                // else add todo to existiong file
+                try{
+                    fs.appendFileSync('todos.txt', `\n${todoText}`);
```

Comment:
it's better to put the `\n` at the end of the line and not before.. 
(Usually files ends with `\n` and it's ok)